deploy/helm/sumologic/templates/_helpers/_common.tpl

```diff
@@ -7,14 +7,15 @@ Expand the name of the chart.
 
 {{/*
 Create a default fully qualified app name.
```

Comment:
I think we want to make the same change for `sumologic.name` above.

Reply:
Is it used anywhere? I grepped for that name and it seems that it's not used anywhere

Reply:
Huh, we should delete it then.

Reply:
I'll do it in another pr